go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/ChainSafeSystems/gossamer/p2p
+
+require (
+	github.com/dgraph-io/badger v1.5.5-0.20190226225317-8115aed38f8f
+	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db
+	github.com/ipfs/go-datastore v0.0.2
+	github.com/ipfs/go-ipfs v0.4.19
```

Comment:
v0.4.19 might not work because it didn't include go module support yet. Try updating to `master ` (`go get github.com/ipfs/go-ipfs@master`) for time being before 0.4.20 is released.